war/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"],
```

Comment:
if your going to use env (which is +1 from me) make sure you have browserlist defined, i'm pretty sure core needs to support at least ie11

Reply:
Will do! Do you think I should make the move to env more visible? I was thinking that it could deserve it's own changelog entry.

Reply:
I kinda want to say delay using babel for a future pr, 

Scratch that.

https://jenkins.io/doc/administration/requirements/web-browsers/ needs to be followed. After that your probably fine. So if your browser list matches up then probably no real concern.

It could potentially break JavaScript for older browsers, but since we don't say we support them, and people using ie10 are unlikely to upgrade Jenkins, I think your fine, but a note certainly wouldn't hurt

Cc @oleg-negashev who updated the list last

Reply:
+1 for following the policy or noting a breaking change if needed

Reply:
I'll write an explanation about the change to babel. Is there any place other than the JIRA ticket and changelog entry where I would need to write about it (or where can I consult the policy).

FWIW, I have tested on a IE VM and it seems to work on IE10, but I agree a disclaimer can be good just in case.

Reply:
possibly here:
https://jenkins.io/doc/developer/building/
or somewhere near here:
https://jenkins.io/doc/developer/views/

Reply:
Some plugins include frontend dev instructions on the package README (https://github.com/jenkinsci/pipeline-stage-view-plugin/tree/master/ui). 

I guess this is not the way that's done on Jenkins core. Are doc pages on jenkins.io preferred?
If this is this the case, I suppose I'd need to create a PR on https://github.com/jenkins-infra/jenkins.io right?

Reply:
jenkins.io would be preferred for docs that affect multiple plugins, developer experience, and guides on how to do things.

if it's very specific to core, it could just be documented in this repo

and yes a PR to that repo if going to jenkins.io

Reply:
I have updated the CONTRIBUTING.md with instructions specific to frontend work: https://github.com/jenkinsci/jenkins/pull/4425/commits/b58ea61223a16868a263ce06f0c76dbc2d7686c0

Reply:
Resolving this one, I'll send a PR to the jenkins.io repo if more docs are needed.